src/TaskLineRenderer.ts

```diff
@@ -267,13 +268,7 @@ function getComponentClassesAndData(component: TaskLayoutComponent, task: Task):
             genericClasses.push(LayoutClasses.description);
             break;
         case 'priority': {
-            let priorityValue = null;
-            if (task.priority === taskModule.Priority.Highest) priorityValue = 'highest';
-            else if (task.priority === taskModule.Priority.High) priorityValue = 'high';
-            else if (task.priority === taskModule.Priority.Medium) priorityValue = 'medium';
-            else if (task.priority === taskModule.Priority.Low) priorityValue = 'low';
-            else if (task.priority === taskModule.Priority.Lowest) priorityValue = 'lowest';
-            else priorityValue = 'normal';
+            const priorityValue = PriorityTools.priorityNameUsingNormal(task.priority).toLocaleLowerCase();
```

Comment:
Nice. Thank you.

This change generates a new warning in WebStorm now, as the code is SOOO simple that it wants `priorityValue` to be inlined:

![image](https://github.com/obsidian-tasks-group/obsidian-tasks/assets/4840096/280fb0c4-5c27-4b78-a784-eddec21230cc)

It's a small thing, but would you mind inlining it, to not grow the number of warnings that will need to be addressed in future?

Reply:
Done. I thought about this at first but the line was too long for my taste =)

Reply:
> Done. I thought about this at first but the line was too long for my taste =)

Understood.

Thanks for changing it